src/crc-start.ts

```diff
@@ -91,12 +92,42 @@ export async function startCrc(
 }
 
 async function askAndStorePullSecret(logger: extensionApi.Logger): Promise<boolean> {
-  const pullSecret = await extensionApi.window.showInputBox({
-    prompt: 'Provide a pull secret',
-    markdownDescription:
-      'To pull container images from the registry, a *pull secret* is necessary. You can get a pull secret from the [Red Hat OpenShift Local download page](https://console.redhat.com/openshift/create/local?sc_cid=7013a000003SUmqAAG). Use the *"Copy pull secret"* option and paste the content into the field above',
-    ignoreFocusOut: true,
-  });
+  let pullSecret: string;
+  const howToPull = await extensionApi.window.showInformationMessage(
+    'To pull container images from the registry, a *pull secret* is necessary. You can login into your Red Hat SSO account to get it configured automatically or manually copy it from browser and paste when requested.',
+    'Sign in with Red Hat SSO',
+    'Configure manually',
+  );
+  if (howToPull) {
+    if (howToPull === 'Use Red Hat SSO Account') {
```

Comment:
This does not seem to match the label as I am always redirected to the manual pull secret window

Reply:
Updated to match